analyses/cell-contamination-removal-analysis/04-find-markers.Rmd

```diff
@@ -199,14 +200,26 @@ table(Idents(seurat_obj))
 
 Seurat can help find markers that define clusters via differential expression (DE). By default, it identifies positive and negative markers of a single cluster (specified in ident.1), compared to all other cells. [FindAllMarkers()](https://satijalab.org/seurat/reference/findallmarkers) automates this process for all clusters, but you can also test groups of clusters vs. each other, or against all cells. 
 
+We save the table with all markers for each cluster here: ./results/02_find_markers/Res_`r {resolution_list}`_Markers_all.tsv.
```

Comment:
I think it would be good for readability to highlight this path in blue, as you did for file paths in the report for the `05_run_summary_report.Rmd` script.